tests/test_instantiate.py

```diff
@@ -25,6 +25,21 @@ def testInstantiate(self):
         mock_args.input_file = None
         mkcc = mkchromecast.Mkchromecast(mock_args)
 
+    def testMP3CodecNodeBackend(self):
+        "This test evaluates the assignment of the MP3 codec when the Node Backend is selected"
```

Comment:
This looks pretty good (and good job including a docstring for this test).  Only change is that the convention is to always use triple-quoted strings for docstrings (so `"""…"""` instead of `"…"`)

(If you're curious, this convention is established in https://peps.python.org/pep-0257/ )